app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
+
+  after_save :update_post_counter
+
+  def return_recent_comments
+    Comment.limit(5).order(created_at: :desc)
+  end

```

Comment:
- [ optional ] You really did a great job but ill recommend using the `order` method before adding the limit because the idea is to order data in descending order before taking the first 5 rows. So ill suggest changing it to something like
 ```
def return_recent_comments
    Comment.order(created_at: :desc).limit(5)
end